eng/Versions.props

```diff
@@ -6,10 +6,11 @@
     <MajorVersion>5</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
-    <!-- Always use shipping version instead of dummy version -->
-    <DotNetUseShippingVersions>true</DotNetUseShippingVersions>
     <PreReleaseVersionLabel>alpha</PreReleaseVersionLabel>
     <PreReleaseVersionIteration>1</PreReleaseVersionIteration>
+
+    <!-- Set assembly version to align with Major and Minor version -->
```

Comment:
Also mention this? My first thought looking at this line is why it doesn't use patch version.

> as for the patches and revisions should be manually updated per assembly if it is serviced

Reply:
Sure. will update the comment.